### ISSUE TITLE
FSR 3 Frame gen dose not work with new RTX HDR in Forza horizon 5 

### ISSUE THREAD

**Issue by Spongypizza1** [Nukem9/dlssg-to-fsr3#396]
I was play forza horizon 5 and fsr 3 frame gen was not working so maybe you can add support for the frame gen and rtx hdr.

**Comment by Macrox14** [comment]
Same with Deliver Us Mars

**Comment by Spongypizza1** [comment]
Yeah assume every game will have the issue

**Comment by Zeryth1** [comment]
FH5 has native HDR why are you using RTX HDR. It's worse and has a performance cost.

**Comment by TimmyP777** [comment]
> FH5 has native HDR why are you using RTX HDR. It's worse and has a performance cost.

Uhhh thats a big negative. This setting improves every HDR implementation Ive ever seen, and producing the best HDR results Ive ever seen.

In games that dont use tensor cores, there is hardly a performance cost. Kingdom Come is a great one.

**Comment by Zeryth1** [comment]
> > FH5 has native HDR why are you using RTX HDR. It's worse and has a performance cost.
> 
> Uhhh thats a big negative. This setting improves every HDR implementation Ive ever seen, and producing the best HDR results Ive ever seen.
> 
> In games that dont use tensor cores, there is hardly a performance cost. Kingdom Come is a great one.

That's not true and generally agreed upon by the experts in the field. I will not argue about this with you.

**Comment by TimmyP777** [comment]
Whats not true? Hundreds of people have already seen my screenshots with native KCD tweaks and RTX HDR, and agreed it looks a lot more next gen... so there is that. Thats enhancing the in game HDR, which obviously can be done since HDR doesn't even have metadata, and even something as simple as Dolby Vision is an improvement on HDR.

In games that dont use tensor cores, there is hardly a performance cost. Kingdom Come is a great one... 

Uhh this is true. Go ahead and turn on RTX HDR in a game that uses DLSS, DLDSR or some RT implementations (probably RR) and see a performance hit, where games that dont use the hardware, don't have a hit. Go ahead and test.

So who are these "experts in the field?"
